### DISCUSSION TITLE
检测模型和识别模型区别

### DISCUSSION THREAD

**Discussion by winnie1228** [PaddlePaddle/PaddleOCR#12687] [Q&A]
分别下载使用中英文超轻量PP-OCRv4模型（15.8M）的检测模型和识别模型过程中，发现返回结果都是一样的，请问这两个模型有什么区别？又该怎么选择使用呢？求回复……
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/78136057/2dc6ebd3-e6ea-46f9-92dd-01666ee72cee)

**Comment by dragronstranding** [comment]
检测模型是检测你文字所在的位置，而识别模型是把你输入图片中的文字识别出来

**Comment by winnie1228** [comment]
> 检测模型是检测你文字所在的位置，而识别模型是把你输入图片中的文字识别出来

感谢回复！但是我在测试过程中两个模型都同时返回了位置和文字内容及概率。如下：[[[[[3.0, 2.0], [801.0, 3.0], [801.0, 42.0], [3.0, 42.0]], ('DAVIDSON DOUGLAS', 0.9832786321640015)], [[[1748.0, 15.0], [2098.0, 15.0], [2098.0, 68.0], [1748.0, 68.0]], ('Semi-t', 0.9988662600517273)], [[[11.0, 44.0], [368.0, 44.0], [368.0, 88.0], [11.0, 88.0]], ('DOS: 06.16.1976', 0.9855455160140991)], [[[1890.0, 99.0], [2118.0, 99.0], [2118.0, 145.0], [1890.0, 145.0]], ('Lortable', 0.9988487958908081)], [[[2022.0, 172.0], [2080.0, 172.0], [2080.0, 238.0], [2022.0, 238.0]], ('P', 0.979322612285614)]]]

**Comment by cuicheng01** [comment]
你是怎么使用的呢？命令是什么呢？

**Comment by HawkingRadiation42** [comment]
the model first uses the detection model which helps in detecting texts from the image and then recognition model reads the texts
